File: javascript/dentro_fora.js
const n = parseInt(prompt("Quantos numeros voce vai digitar?"));

let x, d = 0, f = 0;

for (i = 0; i < n; i++) {
    x = parseInt(prompt("Digite um numero:"));

    if (x >= 10 && x <= 20) {
        d++;
    } else {
        f++;
    }
}

console.log("%d DENTRO\n", d);
console.log("%d FORA\n", f);

File: python/dentro_fora.py
n = int(input("Quantos numeros voce vai digitar? "))

qtd_dentro = 0
qtd_fora = 0

for i in range(0, n):
    valor = int(input("Digite um numero: "))

    if valor >= 10 and valor <= 20:
        qtd_dentro += 1
    else:
        qtd_fora += 1

print(f"{qtd_dentro} DENTRO")
print(f"{qtd_fora} FORA")
